### PR TITLE
Update lesson

### PR DESCRIPTION
**Pull request by jlboba**
Changes made: 
- Configured with TS 
- Added brief intro on passing variables as prop values
- Added instructions for TS specific things (interface, typing props) 

Solution branch available

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,40 @@ by passing it down as a `prop` rather than importing it. This will be
...
+}
+```
+
+works the same as this: 
+
+```jsx
+function Directory() {
+  return(
+    <div>
+      <Person name="Salman" age="27">
```
Comment (lizbur10) [inline README.md]: Change to age={27}

**Review by lizbur10 — CHANGES_REQUESTED** [review]
This looks great - everything runs fine.

File: README.md (hunk 1 above)
Comment (jlboba) [inline README.md]:
```suggestion
      <Person name="Salman" age=27>
```

File: README.md [hunk 2]
```diff
@@ -13,6 +13,40 @@ by passing it down as a `prop` rather than importing it. This will be
...
+}
+```
+
+works the same as this: 
+
+```jsx
+function Directory() {
+  return(
+    <div>
+      <Person name="Salman" age=27>
```
Comment (jlboba) [inline README.md]:
```suggestion
      <Person name="Salman" age={27}>
```

**Pull request merged**